lib/timeline/component/item/Item.js

```diff
@@ -294,6 +294,74 @@ class Item {
     }
   }
 
+	    /**
+	     * Repaint a control button on the top right of the item when the item is selected
+	     * @param {HTMLElement} anchor
+	     * @protected
+	     */
+
+    }, {
+	    key: "_repaintControlButton",
+	    value: function _repaintControlButton(anchor) {
+
+			console.log('_repaintControlButton')
+
+	      var editable = (this.options.editable.overrideItems || this.editable == null) && this.options.editable.control || !this.options.editable.overrideItems && this.editable != null && this.editable.control;
+
+		  console.log('controlButton ' + editable)
+		  console.log('controlButton control option ' + JSON.stringify(this.editable.control))
+
+	      if (this.selected && editable && !this.dom.controlButton) {
+	        // create and show button
+	        var me = this;
+	        var controlButton = document.createElement('div');
+
+	        if (this.options.rtl) {
```

Comment:
`controlButton.className = this.options.rtl ? 'vis-control-rtl' : 'vis-control'`

Reply:
I am not quite sure if I need the rtl option. What does it do?

Reply:
it's a nicer way to write what you wrote with if statements. 
checkout Conditional (Ternary) Operator
https://www.w3schools.com/jsref/jsref_operators.asp